requirements.txt

```diff
@@ -1,6 +1,8 @@
-PySide2
+PySide2; sys_platform != 'darwin' and platform_machine != 'aarch64'
+PySide6; sys_platform == 'darwin' and platform_machine == 'arm64'
```

Comment:
More out of curiosity (and because it triggers my ocd), what's the difference between `arm64` and `aarch64` here?

Reply:
I'm glad it did, because that's a typo - it should be `arm64`! I'll update this now.

The _actual_ difference between `arm64` and `aarch64` is ultimately nothing. `AArch64` is the name of the 64-bit version of ARMv8, `arm64` in regards to Apple tends to be used because (iirc) they had a separate LLVM backend which was for a while dispirate from the `AArch64` one.